base/views.py

```diff
@@ -0,0 +1,4 @@
+"""Purpose of this file
+
+This file describes the frontend views.
```

Comment:
Inwiefern ist dieses File notwendig?
Warum sollen in base/views.py die Frontend views liegen??

Reply:
This file is from the original project and I only added a documentation of the file, so that really (almost) every file in the project is documented. So it is easier to find out what a file is used for or what a function etc. does. See for example in the module `export` the file `views.py`, which has relevant operations related to frontend view.